### PR TITLE
Configure allowed names for Naming/UncommunicativeMethodParamName

### PR DESCRIPTION
**Pull request by jonatas**
Using`AllowedNames` instead of disabling the cop.

Follow up after merging #3.

### Review

- (n/a) Document code according to [Getting Started with Yard](http://www.rubydoc.info/gems/yard/file/docs/GettingStarted.md).
- [x] All tests are passing.
- [x] Test manually.
- [ ] Get approval.

### Pre-merge checklist

- [x] The PR relates to a single subject with a clear title and description in grammatically correct, complete sentences.
- [x] Verify that feature branch is up-to-date with `master` (if not - rebase it).
- [x] Double check the quality of [commit messages](http://chris.beams.io/posts/git-commit/).
- [x] Squash related commits together.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -103,7 +103,10 @@ Naming/HeredocDelimiterNaming:
   Enabled: false
 
 Naming/UncommunicativeMethodParamName:
-  Enabled: false
+  Enabled: true
+  AllowedNames:
+    - as
+    - e
```
Comment (pluff) [inline .rubocop.yml]: `e`? where do we use it? Sounds weird

Comment (bbatsov) [reply .rubocop.yml]: Exception :-)

File: .rubocop.yml [hunk 2]
```diff
@@ -103,7 +103,10 @@ Naming/HeredocDelimiterNaming:
   Enabled: false
 
 Naming/UncommunicativeMethodParamName:
-  Enabled: false
+  Enabled: true
+  AllowedNames:
+    - as
```
Comment (bbatsov) [inline .rubocop.yml]: I guess you can add some explanation about each entry.

File: .rubocop.yml (hunk 1 above)
Comment (jonatas) [reply .rubocop.yml]: But we still have a few of them:
screen shot 2018-03-16 at 20 02 01

File: .rubocop.yml (hunk 2 above)
Comment (jonatas) [reply .rubocop.yml]: Sure thing ✅

**Pull request merged**